specs/serving/knative-api-specification-1.0.md

```diff
@@ -2215,6 +2215,26 @@ Max: 1
    <td><strong>Schema Requirement</strong>
    </td>
   </tr>
+  <tr>
+   <td><code>readOnlyRootFilesystem</code>
+   </td>
+   <td>boolean
+   </td>
+   <td>As specified in Kubernetes <a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#securitycontext-v1-core">core/v1.SecurityContext</a>
+   </td>
+   <td>RECOMMENDED, if securityContext is supported.
```

Comment:
Note that this is part of the [PodSecurityStandards for the `restricted` level](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted)

I think Dave Bailey and Greg Block at Google were interested this back circa 2017, but there was no representation of this property in Knative at the time. Note that this aligns with the runtime spec here: https://github.com/knative/specs/blob/main/specs/serving/runtime-contract.md#root